### DISCUSSION TITLE
Dispersion correction cutoff parsing

### DISCUSSION THREAD

**Discussion by yjw0510** [ACEsuit/mace#459] [Q&A] [answered]
In `foundation_models.py`, the default value for the argument `dispersion_cutoff` is set to `40.0 * units.Bohr`
[mace/mace/calculators/foundations_models.py](https://github.com/ACEsuit/mace/blob/main/mace/calculators/foundations_models.py)
```
def mace_mp(
    model: Union[str, Path] = None,
    device: str = "",
    default_dtype: str = "float32",
    dispersion: bool = False,
    damping: Literal["zero", "bj", "zerom", "bjm"] = "bj",
    dispersion_xc: str = "pbe",
    dispersion_cutoff: float = 40.0 * units.Bohr,
    **kwargs,
) -> MACECalculator:
```

Later in the code, we can see that

```
        d3_calc = TorchDFTD3Calculator(
            device=device,
            damping=damping,
            dtype=dtype,
            xc=dispersion_xc,
            cutoff=dispersion_cutoff,
            **kwargs,
        )
```
However, when I tracked the passed argument in `torch-dftd`, I found that the default value for the cutoff argument is set to `95.0 * Bohr`, while the `cnthr` argument is set to `40.0 * Bohr`.

[torch-dftd/torch_dftd/torch_dftd3_calculator.py](https://github.com/pfnet-research/torch-dftd/blob/1aa73b0c13c973a2ddeeb0366f3c58f3a81c471b/torch_dftd/torch_dftd3_calculator.py#L4)
```
class TorchDFTD3Calculator(Calculator):
...
    def __init__(
        self,
        dft: Optional[Calculator] = None,
        atoms: Atoms = None,
        damping: str = "zero",
        xc: str = "pbe",
        old: bool = False,
        device: str = "cpu",
        cutoff: float = 95.0 * Bohr,
        cnthr: float = 40.0 * Bohr,
        abc: bool = False,
        # --- torch dftd3 specific params ---
        dtype: torch.dtype = torch.float32,
        bidirectional: bool = True,
        cutoff_smoothing: str = "none",
        **kwargs,
    ):
```

Is this simply a mistake confusing `cutoff` for `cnthr`, or is it an intended implementation to decrease the computational load coming from the large cutoff value in the dispersion correction?

**Comment by ilyes319** [comment, marked as answer]
Hey @yjw0510,

Indeed we reduced the cutoff to speed up the computation. We made some limited experiment to select the new default in order to compromise the least the accuracy. More clever approaches could be taken, like adding on top a longer 2 body dispersion, but we have not yet added that.
